core/controllers/profile.py

```diff
@@ -118,11 +118,12 @@ def get(self, _):
     @acl_decorators.is_source_mailchimp
     def post(self, _):
         """Handles POST requests."""
-        if self.request.get('data[list_id]') != feconf.MAILCHIMP_AUDIENCE_ID:
+        if (self.normalized_request.get('data[list_id]')
+            != feconf.MAILCHIMP_AUDIENCE_ID):
```

Comment:
Why do you change this file? The `BulkEmailWebhookEndpoint` does not yet have a schema.

Reply:
Sorry, my bad. Removed them.